src/ripple/app/tx/impl/NFTokenAcceptOffer.cpp

```diff
@@ -121,8 +121,8 @@ NFTokenAcceptOffer::preclaim(PreclaimContext const& ctx)
         // If the buyer specified a destination
         if (auto const dest = bo->at(~sfDestination))
         {
-            // fixUnburnableNFToken
-            if (ctx.view.rules().enabled(fixUnburnableNFToken))
+            // fixNonFungibleTokensV1_2
```

Comment:
what is the value of this comment?

Reply:
yeah this does not seem to add more info in the comment beyond the code itself, ill remove